MapboxNavigationTests/Fixtures/Fixture.swift

```diff
@@ -31,4 +33,22 @@ internal class Fixture {
             return [:]
         }
     }
+    
+    class func downloadRouteFixture(coordinates: [CLLocationCoordinate2D], fileName: String, completion: @escaping () -> Void) {
```

Comment:
We could put this behind an #if to prevent ourselves from checking in tests that call this function.